modules/openxr/openxr_interface.cpp

```diff
@@ -690,6 +690,47 @@ bool OpenXRInterface::set_play_area_mode(XRInterface::PlayAreaMode p_mode) {
 	return false;
 }
 
+PackedVector3Array OpenXRInterface::get_play_area() const {
+	XRServer *xr_server = XRServer::get_singleton();
+	ERR_FAIL_NULL_V(xr_server, PackedVector3Array());
+	PackedVector3Array arr;
+
+	Vector3 sides[4] = {
+		Vector3(-0.5f, 0.0f, -0.5f),
+		Vector3(0.5f, 0.0f, -0.5f),
+		Vector3(0.5f, 0.0f, 0.5f),
+		Vector3(-0.5f, 0.0f, 0.5f),
+	};
+
+	sides[0] = Vector3(-0.5f, 0.0f, -0.5f);
+	sides[1] = Vector3(0.5f, 0.0f, -0.5f);
+	sides[2] = Vector3(0.5f, 0.0f, 0.5f);
+	sides[3] = Vector3(-0.5f, 0.0f, 0.5f);
```

Comment:
Isn't this duplicating the declaration above?

Reply:
Yes it is... weird I overlooked that, we must already have this double in our 3.5 plugin